scripts/send_funds.js

```diff
@@ -1,30 +1,60 @@
 const fs = require("fs");
 const { TxnBatcher } = require("@trustfractal/polkadot-utils");
 const { Keyring, WsProvider, ApiPromise } = require("@polkadot/api");
-const { decodeAddress } = require('@polkadot/keyring');
+const { decodeAddress } = require("@polkadot/keyring");
+const Papa = require("papaparse");
+const cliProgress = require("cli-progress");
+
 const prompt = require("prompt");
-const args = require('minimist')(process.argv.slice(2));
+const args = require("minimist")(process.argv.slice(2));
 
 // Usage:
 //   node send_funds.js \
 //     --amounts send_funds_amounts_example.csv \
-//     --network wss://nodes.testnet.fractalprotocol.com
+//     --out ./sent_txns.csv \
+//     --network wss://nodes.testnet.fractalprotocol.com \
+//     --send-at-once 256
```

Comment:
If we make the comment with /**/ it would be easy to copy the command.

Reply:
True, I don't want to do that right now. I'll remember it for next time I'm working on this file.